build.js

```diff
@@ -13,13 +13,14 @@ const terserOptions =  {
   nameCache: {},
   format: {
     comments: false,
+    ecma: 2015
   },
   mangle: {
     properties: {
       debug: false,
-      keep_quoted: "strict",
       reserved: ['$classData', 'main', 'toString', 'constructor', 'length', 'call', 'apply', 'NaN', 'Infinity', 'undefined'],
-      regex: /^(\$m_|loadHelp|.*__f\d?_|.*__O|.*L\S+_)/,
+      // Basically, every root package except Lcrystal and Lexplore. For some reason it breaks dynamic import of the help system.
```

Comment:
Maybe you need to remove `react` to get the help system?

Reply:
I tried removing them one by one. It's working locally like this.